### DISCUSSION TITLE
[Github Copilot Chat] What is the "Running a second iteration of fix" feature and how can I use it effectively?

### DISCUSSION THREAD

**Discussion by KalleV** [community/community#63006] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I'm seeing Copilot Chat run a 2nd round of changes with a single prompt. It's great, but I don't know how to use it effectively or what triggers it.

**Comment by Akash1134** [comment]
Hi @KalleV ,

I've shared your feedback with the concerned team and will get back to you with more details on the effective use case of "running a second iteration of fix".

Thanks!

**Comment by Akash1134** [comment]
Hi @KalleV ,

Could you please share more details about the message and prompt you're referring to? If you're comfortable sharing it, what was the exact prompt you made that triggered the message? If it is still reproducible, please share a video recording or screenshot of the message you see. Thanks!

**Reply by KalleV** [reply]
Sure, I think I found a consistent way to reproduce it. I selected a big blog of JS code and ran a prompt like "/fix Remove all usage of deprecated prop X". The context messages show up something like:
 - Fetching the response (takes awhile for a big block of code)
 - Applying diagnostics
 - Running second iteration of fix...
 - Diff changes appear
